targets/XPlatCppSdk/source/code/source/playfab/PlayFabEvent.cpp

```diff
@@ -48,7 +48,11 @@ namespace PlayFab
 
     void PlayFabEvent::SetProperty(const std::string& name, const int64_t value)
     {
+#if defined(PLAYFAB_PLATFORM_ANDROID)
+        this->eventContents.Payload[name] = (Int64)value;
+#else // PLAYFAB_PLATFORM_ANDROID
```

Comment:
>// PLAYFAB_PLATFORM_ANDROID [](start = 6, length = 27)

remove since it seems like a little misleading

Reply:
this change is due to different compiler's type conversion on template. In the template, the compiler strictly checking the type and the auto type conversion is different each compiler, and this "long long int" isn't matched any template types. Because JsonCPP didn't implement "long long int" type explicitly.

---
In reply to: [274661025](https://github.com/PlayFab/SDKGenerator/pull/470#discussion_r274661025) [](ancestors = 274661025)